src/utils/material.js

```diff
@@ -127,3 +131,9 @@ function handleTextureEvents (el, texture) {
   });
 }
 module.exports.handleTextureEvents = handleTextureEvents;
+
+module.exports.isHLS = function (videoEl) {
+  if (videoEl.type && videoEl.type.toLowerCase() in ['application/x-mpegurl', 'application/vnd.apple.mpegurl']) { return true; }
+  if (videoEl.src && videoEl.src.toLowerCase().indexOf('.m3u8') > 0) { return true; }
```

Comment:
.ts is not HLS, it is not clear that either .ts playback works or that it needs the workaround (MP4 does not)

if you'd like to add .ts, and it needs this workaround, perhaps you'd like to do in separate PR as this one is merged now?